kyo-prelude/shared/src/main/scala/kyo/Stream.scala

```diff
@@ -43,7 +65,33 @@ sealed abstract class Stream[V, -S]:
     def map[V2, S2](f: V => V2 < S2)(using Tag[Emit[Chunk[V]]], Tag[Emit[Chunk[V2]]], Frame): Stream[V2, S & S2] =
         mapChunk(c => Kyo.foreach(c)(f))
 
-    /** Transforms each chunk in the stream using the given function.
+    /** Transforms each chunk in the stream using the given pure function.
+      *
+      * @param f
+      *   The function to apply to each chunk
+      * @return
+      *   A new stream with transformed chunks
+      */
+    def mapChunk[V2](f: Chunk[V] => Seq[V2])(
```

Comment:
isn't the `NotGiven` needed here?

Reply:
No because since the output is wrapped in `Seq` there's no danger that it will accidentally be a pending value. I include an otherwise unnecessary `Flat[V]` implicit evidence to distinguish the two methods.